fs_storage/models/fs_storage.py

```diff
@@ -148,6 +148,14 @@ def __init__(self, env, ids=(), prefetch_ids=()):
         compute="_compute_options_properties",
         store=False,
     )
+    check_connection_method = fields.Selection(
+        [
+            ("marker_file", "Create Marker file"),
+            ("ls", "List File"),
+        ],
+        default="marker_file",
+        required=True,
+    )
```

Comment:
I put it required because we need a method to check the connection anyway. As there is a default value, it did not seems to be a problem to me.
If we do not put it as required, it means we go with the marker file method if empty and it just seems more hidden for the user.
On my side I don't really care actually

Reply:
A non required field will allow to ignore the check mechanism...

Reply:
That is a good idea. Before the cache issue, the check was not done (it was only with manual button action_test_config) and it was not an issue in my use cases. The fact to check the connection before beeing able to interact with remote server is actually a new "cost" (in time) and it can be nice to make it optional indeed.

The only issue I see, is that in case we set it empty, the manual button `action_test_config` won't work either.
I think it is a shame, you may want to avoid the check mechanism each time you interact the remote server but still be able to easily check all is fine.
Any idea to manage that ? We could force a method in this case but which ?... I'd go with the `ls` one which is less intrusive IMO but not sure it will be ok for every body.

I'll update the PR this way trying to be as clear as I can in the README about the impact of this field.
I guess I'll let the default so stay consistent with current behvior.

Reply:
@florian-dacosta The check button could open a wizard with the same selection field but required....

What we could do:

* define a method on the model to get the possible values for the `check_connection_method`
* modify the  `check_connection_method` field to get possible values from the new method
* define a new wizard to use to test the connection with a `check_connection_method` method field required that will get the possible value from the method on the `fs.storage` model
* replace the logic behind the `action_test_config` method to return a wizard...

What do you thing about?